collections/_article/release-candidate-godot-4-1-rc-3.md

```diff
@@ -0,0 +1,65 @@
+image: /storage/blog/covers/release-candidate-godot-4-1-rc-3.webp
+image_caption_title: "Bravest Coconut"
+image_caption_description: "A game by Nathan Hoad and Lilly Piri"
+date: 2023-07-04 10:00:00
+---
+
+Godot 4.1 is shaping up to be ready [right on schedule](/article/release-management-4-1). Over the course of the beta testing phase, and with our first two [Release Candidates](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate) last week, we identified and fixed a number of regressions, i.e. recently introduced bugs that would worsen the experience compared to Godot 4.0. Now we're fairly confident that we've handled most of those, and the remaining issues we're tracking will likely be deferred to a fix in future 4.1.x maintenance releases.
+
+Our contributors managed to fix a few more critical regressions during the testing phase of RC 2, which justified making a third release candidate to validate those changes. If all goes well, we should be on track for a 4.1 release in a few days. Please keep reporting issues that you encounter with the RC, even if you don't think they're critical – while triaging them, we might prioritize them for early on in the 4.2 development cycle, and backport the fixes to 4.1.x.
+
+This release contains a [number of improvements](/article/dev-snapshot-godot-4-1-beta-1/#highlights) compared to Godot 4.0 published earlier this year. Some systems have also been reworked, which means projects that rely on those need to be updated. However, for most games and apps made with 4.0 it should be safe to migrate to 4.1. Don't forget to always make backups when moving versions, even minor. Better yet, prefer using a version control system, such as Git, and commit a version of your project before the migration.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.1.rc3/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+-----
+
+*The illustration picture for this article is from* **Bravest Coconut**](https://store.steampowered.com/app/2320040/Skies_above_the_Great_War/), *a "classic adventure game featuring a world full of zany characters to meet, spooky monsters to fight, and brain teasing puzzles to solve". The game is developed by Nathan Hoad ([Twitter](https://twitter.com/nathanhoad), [Mastodon](https://mastodon.social/@nathanhoad)) and Lilly Piri ([Twitter](https://twitter.com/lillypiri), [Mastodon](https://mastodon.social/@lillypiri)) using Godot 4. Nathan also makes Godot plugins such as [Dialogue Manager](https://github.com/nathanhoad/godot_dialogue_manager), and publishes devlogs and Godot tutorials on [YouTube](https://www.youtube.com/@nathan_hoad).*
```

Comment:
Why is it in quotation marks? Also the link to the game is wrong :)

Reply:
I copied what Nathan told me in a DM verbatim, and it sounded like the game's elevator pitch. Felt weird to just copy a marketing pitch as is. I'll rephrase it.

Reply:
Fixed.